test/test_fx.py

```diff
@@ -757,8 +758,27 @@ def forward(self, x):
         shape_prop.ShapeProp(tc_traced).propagate(torch.rand(3, 4))
         self.assertEqual(tc_traced.graph.result.shape, ref_out.shape)
 
-
-
+    def test_find_single_partition(self):
+        class testModule(torch.nn.Module):
+            def forward(self, a, b):
+                return a + b
+        m = testModule()
+        traced = symbolic_trace(m)
+        partitioner = Partitioner()
+        devices = [{"name": "dev_0", "available_mem": float('inf')}]
+        dag = partitioner.partition_graph(traced, devices)
+        for node in traced.graph.nodes:
+            assert node.partition_ids == [1]
+        nodes = traced.graph.nodes
+        res_dag = DAG()
+        res_dag.create_node(0, [], [1], [], [])
```

Comment:
To aid in understanding, it might help to use kwargs at the callsite of functions that have many arguments, for example:
```
res_dag.create_node(partition_id=0, parents_ids=[], children_ids=[1], input_nodes=[], output_nodes=[])
res_dag.create_node(partition_id=1, parents_ids=[0], children_ids=[],
                    input_nodes=[nodes[0], nodes[1]], output_nodes=[nodes[2]])
```